velox/expression/SimpleFunctionRegistry.cpp

```diff
@@ -235,16 +235,22 @@ SimpleFunctionRegistry::resolveFunction(
             selectedFunction = currentCandidate.get();
             selectedType = resultType;
             selectedPriority = currentPriority;
-            std::swap(selectedCoercions, requiredCoercions);
+            selectedCoercions = requiredCoercions;
+            selectedCount = 1;
+          } else if (currentPriority == selectedPriority) {
```

Comment:
can you give an example of a case where there is a tie? Is there a test case for this code path?